scraper/main.go

```diff
@@ -47,8 +52,17 @@ func main() {
 
 	models.Verbose = (*verbose == "true")
 	models.Cache = models.NewCache(1000)
+	bearerToken := ""
+
+	if !(*tokenPath == "" || *profile == "" || *region == "") {
+		bearerToken, err = getTokenFromSSM(*tokenPath, *profile, *region)
```

Comment:
maybe we should validate that all of the 3 variables are set.

Reply:
added commands to  'in-built' validate combinations of options and auth modes:
```
creds --user admin --password test --format tf --verbose true -s true
token --region us-west-2 --profile tivo-oi --path /tivooi/itsi_validator_tf_access_token --format tf --verbose true -s true 
```

Reply:
nice. I did not know about parser.NewCommand, that is very useful.